Dynatrace/dynatrace-operator
# Revert digest usage for tenant-registry images

As the tenant-registry is not made to be used with digest, this change goes back to the previous solution of using the build-label on the images of the tenant-registry in order to determine the version, so we can update the oneagent/activegate when necessary.

## How can this be tested?
deploy oneagents/activegates
check if the `image` field only uses the tag, and check that the version label on the statefulset and daemonset is set.

## Checklist
